debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -110,7 +132,24 @@ public static ConnectionFactory patternBasedFactory(String urlPattern, Field...
             String url = findAndReplace(urlPattern, props, varsWithDefaults);
             LOGGER.trace("Props: {}", props);
             LOGGER.trace("URL: {}", url);
-            Connection conn = DriverManager.getConnection(url, props);
+            
+            Connection conn = null;
+            ClassLoader driverClassLoader = classloader;
+            String driverClassName = config.getString(JdbcConfiguration.DRIVER);
+            if (driverClassName == null) {
+                conn = DriverManager.getConnection(url, props);
+            } else {
+                try {
+                    if (driverClassLoader == null) {
+                        driverClassLoader = JdbcConnection.class.getClassLoader();
+                    }
+                    Class<java.sql.Driver> driverClazz = (Class<java.sql.Driver>) Class.forName(driverClassName, true, driverClassLoader);
+                    java.sql.Driver driver = driverClazz.newInstance();
+                    conn = driver.connect(url, props);
+                } catch (ClassNotFoundException|IllegalAccessException|InstantiationException e) {
+                    throw new SQLException(e);
+                }                
+            }
```

Comment:
Yes, agree. We will close this one. I will submit another pull request with those changes. Also, how do you expect my code to call the new pattern based factory? You want me to extend the "MySqlJdbcContext"?